src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public String toString() {
+        return "it's " + getColor() + " " + getNumber() + " ball";
```

Comment:
```suggestion
        return "it's " + color + " " + number + " ball";
```
you have direct access to those fields, no need to use getters